src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -112,87 +112,98 @@ function ScrollDown(props) {
   );
 }
 
-function RetrieveBackgroundImage() {
+async function getDynamicAsset(url, success, failure) {
+  try {
+    const response = await fetch(url);
+
+    if (response.status !== 200) {
+      throw new Error(response.statusText);
+    }
+
+    if (response.headers.get('content-type').includes('application/json')) {
+      success(await response.json());
+    }
+
+    success(response);
+  } catch (error) {
+    console.error('Error getting dynamic asset', error);
+
+    if (failure) {
+      failure();
+    }
+  }
+}
+
+function RetrieveBannerDynamicAssets() {
   const [backgroundImgSrc, setBackgroundImgSrc] = useState('');
   const [transitionBackground, setTransitionBackground] = useState(true);
+  const [stats, setStats] = useState({ stats: { posts: 0, authors: 0, words: 0 } });
+
+  const classes = useStyles();
+  const { telescopeUrl } = useSiteMetadata();
 
   useEffect(() => {
     async function getBackgroundImgSrc() {
-      try {
-        // Uses https://unsplash.com/collections/894/earth-%26-planets collection
-        /* Other Options: 
+      // Uses https://unsplash.com/collections/894/earth-%26-planets collection
+      /* Other Options: 
         - https://unsplash.com/collections/2411320/trend%3A-extreme-neon
         - https://unsplash.com/collections/1538150/milkyway
         - https://unsplash.com/collections/291422/night-lights
         */
 
-        // Ensure we are using an image which fits correctly to user's viewspace
-        const dimensions = `${window.innerWidth}x${window.innerHeight}`;
-        const response = await fetch(`https://source.unsplash.com/collection/894/${dimensions}/`);
-
-        if (response.status !== 200) {
-          throw new Error(response.statusText);
+      // Ensure we are using an image which fits correctly to user's viewspace
+      const dimensions = `${window.innerWidth}x${window.innerHeight}`;
+
+      await getDynamicAsset(
+        `https://source.unsplash.com/collection/894/${dimensions}/`,
```

Comment:
I wonder if we should move this out to the `gatsby-config.js` like we did with `telescopeUrl`?  Don't do it here, we can tweak that in the other PR I have for #894 if necessary.